exploration/percent-format.md

```diff
@@ -135,7 +135,30 @@ _What prior decisions and existing conditions limit the possible design?_
 
 _Describe the proposed solution. Consider syntax, formatting, errors, registry, tooling, interchange._
 
-(Awaiting WG consensus)
+Support the formatting of percent values as follows:
+
+- REQUIRE the `:unit` function for all implementations
+  - Only specific `unit` option values are required, initially the unit `percent`.
+  - The function `:unit unit=percent` does not scale the operand, e.g. `{5 :unit unit=percent}` formats as `5%`.
+- REQUIRE the `:number` and `:integer` functions to support `style=percent` as an option
+  - The functions `:number` and `:integer` scale the operand, e.g. `{5 :integer style=percent}` formats as `500%`.
```

Comment:
I find `:integer style=percent` to be confusing, and I don't think we should support it. Even though it may be well specified, it's not obvious to a reader whether the scaling happens before the rounding, or after it. We should avoid introducing unnecessary uncertainty, esp. as it's likely that implementations will choose to default to `maximumFractionDigits=0` for percent formatting.

Already now, the options supported by `:integer` are a subset of the options supported by `:number`, so this would only extend the set of excluded options.

```suggestion
- REQUIRE the `:number` function to support `style=percent` as an option
  - The function `:number` scales the operand, e.g. `{5 :number style=percent}` formats as `500%`.
```

Reply:
Fair enough. In this commit I was trying to capture the WG telecon conversation and I understood it to include `:integer`.

I don't think `:integer style=percent` is confusing, but it does add more surface to number formatting. Do I suppose correctly that you'd oppose adding other stylistic options (compact, scientific, etc.) to `:integer` as well?

Reply:
Probably, yes. I don't want to commit to the position prematurely, but it'd be my starting point to be a bit queasy about needing to think about whether `4.2e2` is really an "integer" or not.